lib/ntbb-ladder.lib.php

```diff
@@ -158,7 +158,7 @@ function getRating(&$user, $create=false) {
 				}
 				$rp = $this->getrp();
 				$ladderdb->query(
-					"INSERT INTO `{$ladderdb->prefix}ladder` (`formatid`,`userid`,`username`,`rptime`) VALUES (?,?,?,?)",
```

Comment:
@Zarel - Annika and I couldn't figure out why the MySQL on the server was OK without a default value in this `INSERT`? From inspection of the DB, `''` and `0` seem correct here, but i just wanted to call attention to this and ask if maybe you know whats going on? See below for what happens if you run the current query on mysql 5.7 

```
mysql> INSERT INTO `ntbb_ladder` (`formatid`,`userid`,`username`,`rptime`) VALUES ('gen8randombattle','johigufyd','johigufyd','1594198800');
ERROR 1364 (HY000): Field 'rpdata' doesn't have a default value
```

Reply:
I thought it did have a default value. Either way, this seems easy enough to fix.

Reply:
Trying to create a table with default value for `rpdata`  causes an error in MySQL

Reply:
Okay, looks like it's a MEDIUMBLOB in an InnoDB table without a default value; but we're using MySQL 5.5.

It's possible MySQL just lets it go through, or only gives a warning rather than an error.